Mimir.MongoDB/Bson/StakeDocument.cs

```diff
@@ -5,4 +5,4 @@
 namespace Mimir.MongoDB.Bson;
 
 [BsonIgnoreExtraElements]
-public record StakeDocument(Address Address, StakeState Object) : MimirBsonDocument(Address);
+public record StakeDocument(Address Address, Address agentAddress, StakeState Object) : MimirBsonDocument(Address);
```

Comment:
Is it okay if not `AgentAddress`, in the naming convention side?

Reply:
Oops, my bad